contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -876,38 +877,39 @@ <h3 id="capability">Add a Capability to this Contribution</h3>
                                         {% for envvar in capability["deploymentDetails"]["environmentVariables"] %}
                                             {% set keyvalloop = loop %}
                                                 {% if is_editable %}
-                                                    <input class="form-inline"
+                                                    <input class="form-control"
                                                            name="environmentVariables_key_{{keyvalloop.index0}}"
                                                            placeholder="Key"
                                                            id="environmentVariables_key_{{loop.index0}}"
                                                            type="text" value="{{ envvar | filter_nested_dict(["key"]) }}">
-                                                {% else %}
-                                                    <input class="form-inline"
-                                                           name="environmentVariables_key_{{keyvalloop.index0}}"
-                                                           placeholder="Key"
-                                                           id="environmentVariables_key_{{loop.index0}}" type="text">
-                                                {% endif %}
-                                                {% if is_editable %}
-                                                    <input class="form-inline"
+                                                    <input class="form-control"
                                                            name="environmentVariables_value_{{keyvalloop.index0}}"
                                                            placeholder="Value"
                                                            id="environmentVariables_value_{{keyvalloop.index0}}"
-                                                           type="text"
+                                                           type="password"
                                                            value="{{ envvar | filter_nested_dict(["value"]) }}">
+                                                    <i class="bi bi-eye-slash field_icon" id="togglePassword" style="cursor:pointer; position:absolute; margin-left:-20px; margin-top:-20px"></i>
+                                                    <br>
                                                 {% else %}
-                                                    <input class="form-inline"
+                                                    <input class="form-control"
```

Comment:
why the order of environment variable key and value got reversed in here

Reply:
the prev code was like
if editable:
     env_key{{value}}
else:
    env_key
if editable:
    env_value{{value}}
else:
    env_value.

Changed that to : 
if editable:
   env_key{{value}}
   env_value{{value}}
else:
   env_key
   env_value.
This way there is less if-else statements.